modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/css/_admin.scss

```diff
@@ -14,6 +14,39 @@ $toolbarZIndex: 971;
 			font-weight: $font-weight-bold;
 			margin-bottom: 1.5rem;
 		}
+
+		.container-view {
+			padding-bottom: 0;
+			padding-top: 1.5rem;
+		}
+
+		.knowledge-base-child-article-title {
+			font-size: 0.875rem;
+			color: $dark;
+		}
+
+		.list-group-notification .list-group-item {
+			border-width: 1px;
+		}
+
+		.lfr-search-container-wrapper
+			.lfr-search-container-list
+			+ .taglib-search-iterator-page-iterator-bottom {
+			margin-top: 0;
+		}
+
+		.panel-unstyled .panel-body {
+			padding: 1.5rem 0 0.1px;
+		}
+
+		fieldset.knowledge-base-fieldset-margins > legend {
```

Comment:
As mentioned above, remove the new class and use just `fieldset > legend .h3 {...}`, it is much more simple and legible.